### ISSUE TITLE
Suggesting we move away from the template and build a Sanity implementation from scratch

### ISSUE THREAD

**Issue by madisonbullard** [daltn/rave-cafe-blog#9]
The [template we're using](https://vercel.com/templates/next.js/blog-nextjs-sanity) is a lot more code-heavy than a Sanity app needs to be. There is upside, like getting preview and in-place authorship for free, but:
- I'm not sure how much use we'll get out of those features
- We can add them ourselves if we want them
- It would make our app easier to contribute to if we start with just what's needed
- The template employs deprecated patterns

This is stemming from my attempt to migrate to the app router. Among other deprecated patterns, the template uses Next's deprecated "[preview mode](https://nextjs.org/docs/pages/building-your-application/configuring/preview-mode)", but the app router only supports the "[draft mode](https://nextjs.org/docs/app/building-your-application/configuring/draft-mode)" replacement.

The `next-sanity` docs have app router implementation guides, I think this template just hasn't been updated in a while. Things look easy to implement, but a pain to migrate.

I'm proposing we use the templates and docs as reference for a from-scratch implementation of Sanity + Next.

I understand you guys might have work in flight that depends on this template code, so just want to discuss if you agree and if so how to proceed.

**Comment by madisonbullard** [comment]
Oh i also think it'd be more fun to build lol I guess that's a decent reason in projects like this

**Comment by daltn** [comment]
I agree with this assessment of the template and agree that we should build from a clean slate to not build on deprecated patterns. The only in flight work would be @dereksaffe component which we can move over to the new template. This seems to the be the official clean starter - https://github.com/sanity-io/sanity-template-nextjs-clean does this look good to y'all?

**Comment by madisonbullard** [comment]
@daltn yeah that template looks a lot easier to manage 👍🏼 happy to PR the change if you'd like

**Comment by daltn** [comment]
I'll add it to https://github.com/rave-cafe

**Comment by daltn** [comment]
it has been forked - https://github.com/rave-cafe/rave-cafe

**Comment by daltn** [comment]
I'm also open to building a from scratch next js and linking it to sanity if that route ends up being easier vs migrating `/pages` to `/app` on this one.
